File: KingdomCodeTool/wwwroot/Download/202502110928/Service/Implement/NhanVien_User_MappingService.cs
namespace Service_eHospital_DongNai_A_Dictionary.Implement
{
    public class NhanVien_User_MappingService : BaseService<NhanVien_User_Mapping, INhanVien_User_MappingRepository>
    , INhanVien_User_MappingService
    {
    private readonly INhanVien_User_MappingRepository _NhanVien_User_MappingRepository;
    public NhanVien_User_MappingService(INhanVien_User_MappingRepository NhanVien_User_MappingRepository) : base(NhanVien_User_MappingRepository)
    {
    _NhanVien_User_MappingRepository = NhanVien_User_MappingRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202502110928/Angular/shared/NhanVien_User_Mapping.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NhanVien_User_Mapping } from './NhanVien_User_Mapping.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NhanVien_User_MappingService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'NhanVien_User_Id', 'NhanVien_Id', 'User_Id'];

    List: NhanVien_User_Mapping[] | undefined;
    ListFilter: NhanVien_User_Mapping[] | undefined;
    FormData!: NhanVien_User_Mapping;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NhanVien_User_Mapping";
    }
}

